packages/vscode-extension/src/utilities/getMinimumSupportedNodeVersion.ts

```diff
@@ -8,8 +11,10 @@ of react native, so it should be always used after node_modules installation.
 */
 export function getMinimumSupportedNodeVersion(appRoot: string): string {
   const appPackageJson = require(`${appRoot}/package.json`);
+  let applicationMinimumNodeVersion;
```

Comment:
Isn't this only assigned once? Also, you should be able to use the nullable member access operator.

```suggestion
  const applicationMinimumNodeVersion = appPackageJson?.engines?.node;
```